demos/index.html

```diff
@@ -62,7 +72,28 @@
                                 width: 400,
                                 height: 400
                             }
-                        }
```

Comment:
The options block for instantiating "floe.chartAuthoring.demo" is getting longer and scary. Might be nice to rearrange some of those. I'm thinking options like dataEntryPanel.strings, pieChart.pieChartOptions.xxxDisplayTemplate can be moved into components where they are defined. Other options like for templateLoader, pieChart.pieChartOptions.colors and initialData can be moved to chartAuthoring-demo.js. Let me know if it makes sense.